wire-ios/Wire-iOS/Resources/Base.lproj/Localizable.strings

```diff
@@ -429,6 +429,31 @@
 
 "content.system.services.warning" = "Services have access to the content of this conversation";
 
+"content.system.missing_messages" = "You haven’t used this device for a while. Some messages may not appear here.";
```

Comment:
you can run the crowdin script to get the keys up for translations

Reply:
I will do that.